specification/resources/resource-manager/Microsoft.Resources/stable/2025-04-01/resources.json

```diff
@@ -6596,6 +6596,10 @@
           "type": "object",
           "description": "A subset of properties that uniquely identify a Bicep extensible resource because it lacks a resource id like an Azure resource has."
         },
+        "extension": {
+          "$ref": "#/definitions/DeploymentExtensionDefinition",
+          "description": "The key of the extension the resource was deployed with."
```

Comment:
Btw this is not a key, but an object. I think we can just say `The extension the resource was deployed with.`?

Reply:
Looks like it's copied from https://github.com/Azure/azure-rest-api-specs/blob/741dbe9e929847803db78dfb6b094bc813640f70/specification/resources/resource-manager/Microsoft.Resources/stable/2025-04-01/resources.json#L6601, which might need an update as well.

Reply:
Good point. This was a copy-paste. I've updated the place I copied it from.